index.js

```diff
@@ -184,7 +186,7 @@ async function httpProxy (fastify, opts) {
   })
 
   function handler (request, reply) {
-    let dest = request.raw.url
+    let dest = request.urlData().path
```

Comment:
Isn't it like in `fastify` the query param is placed anyways in `request: FastifyRequest` under `query`? 
In my case query param didn't disappear and was placed to redirected address. 
This should not be breaking change as any query param in the url simply broke app and gave `500`.

Probably if some users want to skip query param it could be done in `preHandler`?